MagaluCloud/magalu
# fix(schema): fix conversion ref resolution, fix recursive simplification

## Description

Fix schema recursion when simplifying and converting, and resolve all refs when converting

## How to test it

For now, look at the help output diffs in this PR. And also, run `mgc config list` (which is known to have a recursive schema` to see if everything works
